tests/std/tests/GH_000639_nvcc_include_all/env.lst

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation.

```

Comment:
Don't we need to add this test to `tests/std/tests.lst`? How do we keep it from running internally? (Or does it now work internally despite that `nvcc` is 9.2?)

Reply:
This test is not meant to run internally. This PR changes the mechanism by which we discover tests to just be any directory under `std/tests`.

Reply:
> This PR changes the mechanism by which we discover tests to just be any directory under `std/tests`.

The description should probably mention that.